assets/src/components/card/style.scss

```diff
@@ -0,0 +1,12 @@
+/**
+ * Card styles.
+ */
+
+.newspack-card {
+	background: $muriel-white;
+	border-radius: 3px;
+	box-shadow: 0px 1px 1px rgba(0, 0, 0, 0.25);
+	margin-bottom: 10px;
+	max-width: 440px;
+	padding: 24px;
+}
```

Comment:
I think it may be good to add `margin-left: auto; margin-right: auto`. Currently cards are left-aligned, and we want them centered.

Reply:
Done in https://github.com/Automattic/newspack-plugin/pull/28/commits/c6231e5b5c29fe206c18e1c1126a3e34eba3bdb8.